include/NotificationsManager.php

```diff
@@ -720,10 +720,12 @@ public function introNotifs($all = false, $start = 0, $limit = 80) {
 			$sql_extra = " AND `ignore` = 0 ";
 
 		/// @todo Fetch contact details by "get_contact_details_by_url" instead of queries to contact, fcontact and gcontact
```

Comment:
Yes, you are right, but I would like to change this in the next release. This could introduce possible bugs and at the moment it works good this way